pkgs/development/python-modules/cylp/default.nix

```diff
@@ -0,0 +1,73 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, numpy
+, scipy
+, python
+, pytestCheckHook
+, cython
+, cbc
+, bzip2
+, zlib
+, pkg-config
+}:
+
+buildPythonPackage rec {
+  pname = "cylp";
+  version = "0.91.4";
+
+  src = fetchFromGitHub {
+    owner = "coin-or";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-m09s+QrkLGtF83tDLNjU4ECOqgW5RT0h9+fSwDeADag=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    cython
+  ];
+
+  buildInputs = [
+    cbc
+    bzip2
+    zlib
+  ];
+
+  propagatedBuildInputs = [
+    numpy
+    scipy
+  ];
+
+  preBuild = ''
+    # Leak some memory, reverting this commit. Otherwise there's a segfault
+    # that's being debugged in https://github.com/coin-or/CyLP/issues/105
+    substituteInPlace cylp/cy/CyClpSimplex.pyx --replace \
+      'del self.CppSelf' \
+      'pass'
+
+    # Rebuild all cython files
```

Comment:
Isn't that implicit in the Nix build process? Comment doesn't seem useful.

Reply:
I wrote a bit more in the comment about why this is a good idea. They ship codegen'd  C++ files on github, but for us its better to regenerate them.